src/app/admin/components/courthouses/courthouse-search-results/courthouse-search-results.component.ts

```diff
@@ -15,6 +15,7 @@ import { TableRowTemplateDirective } from '@directives/table-row-template.direct
 })
 export class CourthouseSearchResultsComponent {
   @Input() results: Courthouse[] = [];
+  @Input() show!: boolean;
```

Comment:
`@Input() show = false;`

It's better to give a default value and drop the type for a boolean prop like this